Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+# Aplicatii web pentru incepatori
+######  Autor : Oancea Ionut Eugen
+
+![HACKED.io: Hands on FirefoxOS](https://arcturus.github.io/firefoxos-hackedio/img/html5_css_javascript.png)
+
+Orice aplicatie web acesibila intr-un browser foloseste o combinatie de 3 limbaje pentru a ajunge la forma sa finala (HTML + CSS + JavaScript). 
+Pentru a continua ne este indeajuns sa intelegem pe scurt rolul fiecarui limbaj intr-o aplicatie
+Nu ne vom arunca prea mult inca in detalii la acest punct.
+
+- *Am adaugat un link catre wikipedia cu niste explicatii, pentru cei care doresc insa acest lucru.*
+
+#### - HTML ( Hyper text markup language )
+[HTML Wikipedia](https://ro.wikipedia.org/wiki/HyperText_Markup_Language)
+Este folosit pentru a crea pagini web simple ce pot fi afișate într-un browser.
+Scopul HTML este  in general prezentarea informațiilor – paragrafe fonturi, tabele ș.a.m.d si este limitat la acest lucru de unul singur.
+
+#### - CSS ( Cascading Style Sheets )
+[CSS Wikipedia](https://ro.wikipedia.org/wiki/Cascading_Style_Sheets)
+ CSS formateaza HTML-ul scris, si cu ajutorul unui set de prorietati se pot pozitiona elemente in pagina, se pot face animatii, ș.a.m.d
+
```

Comment:
Mi-e frica ca explicatiile astea sa nu fie prea "tehnice" pentru un absolut incepator. Ai putea incerca sa le rescrii, cu cat mai putine detalii si incercand sa te raportezi la persoane complet incepatoare. Ceva de genul

`CSS este folosit pentru stilizarea paginilor Web. Este un limbaj extrem de puternic, cu care putem crea niste pagini cu adevarat WOW`. Iar acum ca i-am facut curiosi, poate gasesti exemple (CodePen, sau pagini Web de sine statatoare) dar cu stiluri foarte faine ca sa le crestem entuziasmul. Poti sa arunci un ochi aici, am facut ceva similar pentru studenti: http://iampava.com/half-time-fe/

Reply:
M-am gandit oarecum sa pastrez o balanta aici.
Pana la urma cineva poate sa fuga de detalii, daca i se pare prea complicat.

Am sa incerc sam merg pe un still friendly, `Hey look what it can do :D` si am sa las sa se experimenteze cu exercitii interesante